packages/block-editor/src/components/inserter/menu.js

```diff
@@ -356,7 +358,33 @@ export class InserterMenu extends Component {
 							</PanelBody>
 						) }
 						{ isEmpty( suggestedItems ) && isEmpty( reusableItems ) && isEmpty( itemsPerCategory ) && (
-							<p className="editor-inserter__no-results block-editor-inserter__no-results">{ __( 'No blocks found.' ) }</p>
+							<p className="editor-inserter__no-results block-editor-inserter__no-results">
+								{ numberOfHiddenBlocks ? __( 'No active blocks found.' ) : __( 'No blocks found.' ) }
+								{ !! numberOfHiddenBlocks && (
+									<>
+										<br />
+										{
+											sprintf(
+												_n(
+													'%1$d block is disabled.',
+													'%1$d blocks are disabled.',
+													numberOfHiddenBlocks
+												),
+												numberOfHiddenBlocks
+											)
+										}
+										<br />
+										<Button
+											isButton
```

Comment:
There is no „isButton” prop in „Button” component.

Reply:
🤦‍♂ haha, I suppose not, removed. (I was trying to get it styled like a button and tried that before I added isDefault and forgot to remove it)